README.md

```diff
@@ -8,21 +8,31 @@ for production deployment and full stack dev env info.
 
 For development, to run just this stack, do 
 ```bash
+$EDITOR /etc/hosts
+# set an entry like
+# 127.0.0.1 wetfishonline.com.local
+
+# setup env files
 cp mariadb.env.example mariadb.env
 # -> edit, change passwords and other info as needed
 cp php.env.example php.env
 # -> edit, change passwords to match
 
+# ensure www-data user inside container can write to persistent storage
+sudo chown -R 33:33 storage/fish
```

Comment:
Will this command need to also be manually run on production? Do you have a different approach for handling fish storage permissions on prod?

Reply:
It will, same approach for prod, although switching to named volumes is easy if you'd prefer.